tripy/tripy/frontend/module/module.py

```diff
@@ -97,6 +97,8 @@ def __call__(self, x):
 
     def __setattr__(self, name: str, value: Any) -> None:
         if isinstance(value, Parameter) or name in dict(self.named_parameters()):
+            if not isinstance(value, Parameter):
+                value = Parameter(value)
```

Comment:
Will this affect non-Parameter attrs? E.g. a simple bool option `self.multi_output = True`.

Reply:
No, because it's guarded by the `name in dict(self.named_parameters())` check